pkg/machinepool/autoscaling.go

```diff
@@ -0,0 +1,41 @@
+package machinepool
+
+import (
+	"fmt"
+	"github.com/giantswarm/apiextensions/v3/pkg/annotation"
+	"github.com/giantswarm/azure-admission-controller/pkg/mutator"
+	capiexp "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
+	"strings"
+)
+
+// Ensure the needed escapes are in place. See https://tools.ietf.org/html/rfc6901#section-3 .
+func escapeJSONPatchString(input string) string {
+	input = strings.ReplaceAll(input, "~", "~0")
+	input = strings.ReplaceAll(input, "/", "~1")
+
+	return input
+}
+
+// setDefaultSpecValues checks if some optional field is not set, and sets
```

Comment:
At least the comment function name is out-of-date, but maybe the body could be more specific as well? Alternatively just drop it.

Reply:
fixed, thanks